core-blocks/paragraph/edit.native.js

```diff
@@ -0,0 +1,65 @@
+import { View } from 'react-native';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import {
+	Component,
+} from '@wordpress/element';
+import { parse } from '@wordpress/blocks';
+
+import { RichText } from '@wordpress/editor';
+
+const _minHeight = 50;
+
+class ParagraphBlock extends Component {
+	render() {
+		const {
+			attributes,
+			setAttributes,
+			//className,
+			style,
+		} = this.props;
+
+		const {
+			placeholder,
+		} = attributes;
+
+		return (
+			<View>
+				<RichText
+					content={ { contentTree: attributes.content, eventCount: attributes.eventCount } }
+					style={ style, [
```

Comment:
This `style` definition is broken. Prop can take only one value so it probably doesn't take `minHeight` into account.